lib/EasyPost/EasypostResource.php

```diff
@@ -65,11 +65,14 @@ public function instanceUrl()
     /**
      * Refresh the object from the API.
      *
+     * @deprecated This function will be removed in the next major version.
```

Comment:
I don't want to lead to confusion.
```suggestion
     * @deprecated This function will be made unavailable in the next major version.
```